tsconfig.json

```diff
@@ -1,29 +1,11 @@
 {
-	"version": "2.1.5",
+	"extends": "./node_modules/@dojo/scripts/tsconfig/umd.json",
 	"compilerOptions": {
 		"declaration": false,
```

Comment:
whats the default for this?

Reply:
It defaults to true, probably because it is based off the shim/core tsconfig.json which both are set to true.